### ISSUE TITLE
Bursa aspiration: positive gram stain

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10819]
**Preferred term label:**
Bursa aspiration: positive gram stain

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Prepatellar bursitis
Olecranon bursitis

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001434
- label: Positive bursa fluid gram stain
- parents: [Positive infectious agent test (HP:0430030)](https://hpo.jax.org/app/browse/term/HP:0430030)
- pmid: 
- def: Demonstration of the presence of bacteria by gram staining of bursa fluid. The sample is generally obtained by bursal fluid aspiration.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10819
- synonyms:
